src/Http/Routing/ref/Microsoft.AspNetCore.Routing.netcoreapp3.0.cs

```diff
@@ -490,71 +490,19 @@ public partial class StringRouteConstraint : Microsoft.AspNetCore.Routing.IParam
 }
 namespace Microsoft.AspNetCore.Routing.Internal
 {
-    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public readonly partial struct BufferValue
-    {
-        private readonly object _dummy;
-        private readonly int _dummyPrimitive;
-        public BufferValue(string value, bool requiresEncoding) { throw null; }
-        public bool RequiresEncoding { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
-        public string Value { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
-    }
     public partial class DfaGraphWriter
```

Comment:
Should this be `internal`? Is there a good reason to have this be part of reference assemblies?

Reply:
Read the comments on the class. This is intentionally pubternal.